Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Use slim Python base image
+FROM python:3.11-slim
+
+# Set working directory
+WORKDIR /app
+
+# Install Git
+RUN apt-get update && apt-get install -y git && rm -rf /var/lib/apt/lists/*
+
+# Copy requirements and install
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy model
+COPY model.joblib .
```

Comment:
I don't think we should embed the model in the image. It's an artifact of model-training so it should be pushed to a publicly accessible URI (GDrive or maybe One Drive)